tests/cypress/tests/server/copy_paste_test.ts

```diff
@@ -1,10 +1,11 @@
-import DashTable from 'cypress/DashTable';
+import DashTable, { State } from 'cypress/DashTable';
 import DOM from 'cypress/DOM';
 import Key from 'cypress/Key';
 
 describe('copy paste', () => {
     beforeEach(() => {
         cy.visit('http://localhost:8082');
+        cy.wait(1000);
```

Comment:
There's a double loading loop for data as input and state. Give it some time just in case.